packages/driver/cypress/e2e/cypress/stack_utils.cy.js

```diff
@@ -23,6 +23,39 @@ describe('driver/src/cypress/stack_utils', () => {
     })
   })
 
+  context('getRelativePathFromRoot', () => {
+    const relativeFile = 'relative/path/to/file.js'
+    const absoluteFile = 'User/ruby/cypress/packages/driver/relative/path/to/file.js'
+    const repoRoot = 'User/ruby/cypress'
+    const relativePathFromRoot = 'packages/driver/relative/path/to/file.js'
+
+    it('returns relativeFile if absoluteFile is empty', () => {
+      const result = $stackUtils.getRelativePathFromRoot(relativeFile, undefined)
+
+      expect(result).to.equal(relativeFile)
+    })
+
+    it('returns relativeFile if `repoRoot` is not set in the config', () => {
+      const result = $stackUtils.getRelativePathFromRoot(relativeFile, absoluteFile)
+
+      expect(result).to.equal(relativeFile)
+    })
+
+    it('returns relativeFile if absoluteFile does not start with `repoRoot`', () => {
+      Cypress.config('repoRoot', 'User/ruby/test-repo')
```

Comment:
This one isn't working see comment below

Reply:
I don't know how to fix it just by looking at it, but I can pull it down and find out why

Reply:
This one is passing for me 

![image](https://user-images.githubusercontent.com/19196536/190595535-11bf1630-606b-4e7e-8daa-6572d56422a4.png)

Reply:
If I do `context.only('getRelativePathFromRoot')` this is fine, but then if you also add `  context.only('.getCodeFrame', () => {` it fails. These two tests have some kind impact on each other.

Reply:
I think https://github.com/cypress-io/cypress/pull/23833/files#diff-430e30e905812f3055196475c7f8d091ec640ccf95dec5e52dae9799204674daR130 is the problematic one, you've mistakenly called out this one since they both have the same test description. Line 130 is the failing spec.